sBTC SIP

```diff
@@ -0,0 +1,166 @@
+2. **Withdrawal accept:** If accepted, the following happens:
+    - The signers create a "Withdrawal accept" transaction on Bitcoin which returns the requested amount to the designated address.
+    - Once the Bitcoin transaction is confirmed, the signers must then call into the sBTC protocol to mark it as complete - this is done through a Stacks transaction.
+    - If successful, the resulting Stacks transaction will record the withdrawal request as complete & will accordingly burn the user’s locked-sBTC.
+3. **Withdrawal reject:** If instead the request is rejected, the sBTC signers will call the withdraw-reject function in the .sbtc smart contract. This function does the following:
+    - Returns the sBTC to the holder.
+    - Records the signer votes.
+
+To support deposits and withdrawals in sBTC, Signers must collectively make smart contract calls on Stacks. We require each signer to individually sign any contract call and ensure at least 70% of the signer signatures are present for each contract call to be valid.
+
+## Auxiliary Features
+Auxiliary features of the sBTC Bootstrap protocol are described below.
+
+- **Stacks Transaction Fee Sponsorship:** sBTC will include the option to have sBTC transactions on Stacks be sponsored in return for some sBTC. Using the approach suggested in this issue, sBTC users will be able to nearly spend sBTC as gas by getting support from an existing STX holder.
+- **Signer Wallet Rotation:** Mechanisms are provided for the scenario where a signer wants to rotate their key. For this to happen, signers must coordinate offline and vote on-chain on the new signer set (aka set of keys). Once the new signer set is determined, the signers conduct a wallet handoff and re-execute DKG.
+
+### sBTC Bootstrap Signer Eligibility
+The Stacks 2.5 epoch activated the Stacks Signer network, whereby Stackers (users who lock STX in consensus in order to earn BTC rewards) are able to participate in Stacks consensus. Stackers vote to append blocks to the Stacks blockchain by producing a signature over the block. Each Stacker's share of the signature (its weight) is proportional to the fraction of locked STX it owns. This essential function ensures the security and liveness of the Stacks network.
+
+The sBTC Bootstrap Signers will be a separate subset of Stacks Signers. For the sBTC Bootstrap release, Signers will run a binary in addition to the core Stacks software. There will be 15 signers participating in the sBTC bootstrap release. In a subsequent release and hard fork, sBTC will be enshrined in Stacks consensus and Signers will only need to maintain a single binary.
+
+The following eligibility criteria has been used to identify the sBTC Bootstrap Signers:
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet for at least two previous cycles.
```

Comment:
How do you prove this?  Stackers don't do anything with their signing keys in Stacks 2.5; they're only required to register them in order to boot up 3.0.  Are you thinking of 3.0 instead?